earth_enterprise/src/fusion/autoingest/sysman/SConscript

```diff
@@ -108,7 +108,7 @@ gesystemmanager = env.executable('gesystemmanager',
       env['FUSIONVER'],
       ] +
     env.ObjFromOtherDir(extraObjs),
-    LIBS=['gegdal', 'geraster', 'gesearchtabs', 'genet', 'gepublish',
+    LIBS=['gegdal', 'geraster', 'gesearchtabs', 'genet', 'gepublish', 'geconfigutil','gefconfigutil',
```

Comment:
If possible, remove `geconfigutil` from here - I think you only need `gefconfigutil`.

Reply:
done